elements/pfe-progress-steps/src/pfe-progress-steps-item.scss

```diff
@@ -41,7 +41,7 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
 $LOCAL: progress-steps;
 :host(:not([vertical])) {
   width: pfe-local(size, $region: item);
-  min-width: fit-content;
+  text-align: center;
```

Comment:
The step items only line up correctly if the width is explicitly set.  fit-content was messing with it.